app/javascript/components/layout/CustomTable.js

```diff
@@ -185,7 +185,7 @@ class CustomTable extends React.Component {
         {this.props.rowData.length === 0 && <div className="text-center">No data available in table.</div>}
         <div className="d-flex justify-content-between">
           <Form inline className="align-middle">
-            <Row>
+            <Row style={{ width: '430px' }}>
```

Comment:
I think this is a reasonable approach to fixing this issue.

Question: why 430? could we get away with something smaller? I would say as long as that can "fit" four digits, it's big enough

Also, please add this as a new CSS class in here https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/packs/stylesheets/custom_table.scss

Reply:
I came to 430 through trial and error after testing the size across all browsers on both my PC and Mac. If it goes much smaller, it doesn't render properly on Chrome and Firefox. It only appears more stretched out than necessary on IE11 with this value. 

I will make the adjustment you recommended, thanks!